.storybook/recipes/Filters/Filters.tsx

```diff
@@ -37,10 +36,167 @@ export const Filters = ({ placement, ...other }: Props) => {
       setIsLarge(false);
     }
   }, 200);
+
+  const [checked, setChecked] = useState({
+    salad: false,
+    sandwich: false,
+    soup: false,
+  });
+
+  const onCheckboxChange = (food: string) => {
+    const newChecked = { ...checked };
+    newChecked[food] = !checked[food];
+    setChecked(newChecked);
+  };
+
+  /**
+   * Counts how many filters have been applied.
+   */
+  const filterCount = Object.values(checked).reduce((count: number, status) => {
+    if (status) return count + 1;
+    else return count;
```

Comment:
Could this be collapsed into a ternary?
```suggestion
    return status ? count + 1 : count;
```

Reply:
Def could, thanks